Testing/Testing - Battery Usage.md

```diff
@@ -0,0 +1,69 @@
+# Android - Profiler
+
+This tool helps you find where your app uses more energy than necessary.  
+It consists of three timelines: Event timeline, Energy timeline and System timeline.  
+Profiler monitors use of CPU, network, GPS and system events and presents them in form of bars and horizontal lines (system events).  
+Higher/longer the bar is - more energy demanding that part is.  
+You can place mouse over any part of timeline for more info.
+
+ ![1_battery.png](/img/1_battery.png)
+
+
+## Prerequisites:
+###Developer options
+1.  Go to Software info (Location varies depending on phone brand), 
+2.  Tap multiple 	times on "Build Number" (or "MIUI version") till you see toast 	message: "You are "X" steps away from being a developer."  
+3. Enable "Developer options"  
+Xiaomi/Realme - Settings, Additional settings  
+Huawei/Samsung - Settings  
+LG - System
```

Comment:
Ovo čak ne moramo navoditi. Super je info, ali nekako miče fokus s glavnih koraka. Možeš čak kod točke 3 ponoviti zagradu koji imaš kod točke 1: "(Location varies depending on the phone brand)"